features/step_definitions/request.rb

```diff
@@ -42,12 +43,40 @@ def fixtures
       "unique_lower": u.downcase,
       "unique_alnum": alnum,
       "unique_lower_alnum": alnum.downcase,
-      "now_ts": Time.now.to_i,
-      "now_iso": Time.now.iso8601,
-      "hour_later_ts": (Time.now + 3600).to_i,
-      "hour_later_iso": (Time.now + 3600).iso8601,
-      "hour_ago_ts": (Time.now - 3600).to_i,
-      "hour_ago_iso": (Time.now - 3600).iso8601,
+      "now": Time.now,
```

Comment:
Same do we need that one?

Reply:
I'm using it line 55, but maybe I can use the `Time.now` directly